### PR TITLE
28 add media linking to user accounts

### PR DESCRIPTION
**Pull request by IgalTam**
There might be some eslint/prettier issues, these can be ignored until we address them in the overall codebase.

### CODE REVIEW

**Comment by IgalTam** [comment]
Need to add ```package-lock.json``` back into ```express-backend/``` to fix backend dependency error.

File: express-backend/models/user-services.js [hunk 1]
```diff
@@ -52,15 +54,34 @@ exports.addUser = async function addUser(user) {
...
+    }
+    const newMedia = patchObj['media_list'];
+    if (newMedia) {
+      // add/remove media document references
+      for (let i = 0; i < newMedia.length; i++) {
+        if (userToPatch.media_list.includes(newMedia[i])) {
+          // remove existing media document reference
+          const index = userToPatch.media_list.indexOf(newMedia[i]);
+          if (index > -1) {
+            userToPatch.media_list.splice(index, 1);
```
Comment (crobin27) [inline express-backend/models/user-services.js:78]: Glad you were able to get the patch working, I'll be sure to test this when I get to updating the showInformation page for current episode. Everything else LGTM.

**Review by crobin27 — COMMENTED** [review]
Great work solidifying the backend Igal, frontend link is slowly coming together and I've been making use of the post and get commands.

**Review by mpicazocp — APPROVED** [review]

**Comment by IgalTam** [comment]
Thank you all for the approval. I believe that the issues causing the Frontend CI have been fixed, so I will merge this branch into main separately from this PR. Also, I will be updating the User and Media schema to better reflect our current frontend operations in the near future, as me and @SamuelSehnert discussed. As such, it may be good for you all to be open to updating your respective frontend sections in the event of compatibility collisions.

**Pull request merged**